docker/docker-compose.rococo.yml

```diff
@@ -5,9 +5,9 @@ services:
       service: faucet
     environment:
       SMF_BACKEND_NETWORK_DECIMALS: 12
-      SMF_BACKEND_RPC_ENDPOINT: "wss://rococo-rpc.polkadot.io/"
+      SMF_BACKEND_RPC_ENDPOINT: "wss://rpc.rococo.frequency.xyz/"
 
       SMF_BOT_MATRIX_SERVER: https://m.parity.io
       SMF_BOT_MATRIX_BOT_USER_ID: '@rococo-faucet-bot:matrix.org'
-      SMF_BOT_DRIP_AMOUNT: 100
-      SMF_BOT_NETWORK_UNIT: ROC
+      SMF_BOT_DRIP_AMOUNT: "0.5"
```

Comment:
Should this be 5000?

Reply:
it translates to 5000 tokens. It does some multiplication inside somewhere

Reply:
You are right that might be wrong, since I have not tested in bot I ca not confirm. Let me check

Reply:
Can't find any usages in the codebase which is weird but I'll change it to 5000 just in case